frontend/talentsearch/src/js/components/Browse/OngoingRecruitmentSection.tsx

```diff
@@ -619,75 +644,128 @@ export const OngoingRecruitmentSection = ({
             "instructions for section with ongoing pool advertisements",
         })}
       </p>
-      <div data-h2-padding="base(x2, 0, 0, 0) p-tablet(x3, 0, 0, 0)">
+      <div data-h2-display="base(flex)">
+        <FormProvider {...methods}>
+          <form>
+            <Select
+              id="quickFilter"
+              name="quickFilter"
+              label={intl.formatMessage({
+                defaultMessage: "Quick filter",
+                id: "8NB+Ay",
+                description: "A label for a quick filter input.",
+              })}
+              options={[
+                {
+                  value: "",
+                  disabled: true,
+                  label: intl.formatMessage({
+                    defaultMessage: "Select a job stream...",
+                    id: "cmFeXj",
+                    description:
+                      "Placeholder for stream filter in browse opportunities form.",
+                  }),
+                },
+                ...streamsWithAvailablePools.map((stream) => ({
+                  value: stream.key,
+                  label: stream.title,
+                })),
+              ]}
+              trackUnsaved={false}
+            />
+          </form>
+        </FormProvider>
+      </div>
+
+      <p
+        aria-live="polite"
+        aria-atomic="true"
```

Comment:
I made a mistake when I recommended using `aria-atomic=true`, it results in a poorer UX for screen readers. Sorry about that! 😢 

```suggestion
```

Reply:
It only announces the content that has changed so it just sounds like it is repeating the option you selected. I thought it would be better but it just sounds odd and the other text isn't that verbose so dropping it seems like a good option.